array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+
+function last(array) {
+  if (array.length > 0) {
```

Comment:
If the array length is 0, wouldn't the value of the `array[array.length - 1]` be `undefined` anyway?